selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/FilterOrganizationTest.java

```diff
@@ -25,82 +27,81 @@
 import org.eclipse.che.selenium.pageobject.dashboard.organization.AddOrganization;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationListPage;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationPage;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /**
- * Test validates organization creation and actions on it in the list of organizations.
+ * Test validates organization filter
  *
  * @author Ann Shumilova
  */
 public class FilterOrganizationTest {
-  private static final Logger LOG = LoggerFactory.getLogger(FilterOrganizationTest.class);
+  private static final String ORGANIZATION_NAME = generate("organization", 5);
 
   private List<OrganizationDto> organizations;
-  private String organizationName;
-
-  @Inject private OrganizationListPage organizationListPage;
-  @Inject private OrganizationPage organizationPage;
-  @Inject private NavigationBar navigationBar;
-  @Inject private Dashboard dashboard;
-  @Inject private AddOrganization addOrganization;
 
   @Inject
   @Named("admin")
   private TestOrganizationServiceClient testOrganizationServiceClient;
 
+  @Inject private OrganizationListPage organizationListPage;
+  @Inject private OrganizationPage organizationPage;
+  @Inject private AddOrganization addOrganization;
+  @Inject private NavigationBar navigationBar;
   @Inject private AdminTestUser adminTestUser;
+  @Inject private Dashboard dashboard;
 
   @BeforeClass
   public void setUp() throws Exception {
-    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
-
-    organizationName = NameGenerator.generate("organization", 5);
+    testOrganizationServiceClient.create(ORGANIZATION_NAME);
+    testOrganizationServiceClient.create(generate("organization", 7));
+    testOrganizationServiceClient.create(generate("organization", 7));
+    testOrganizationServiceClient.create(generate("organization", 7));
     organizations = testOrganizationServiceClient.getAll();
+
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteByName(organizationName);
+    for (OrganizationDto organization : testOrganizationServiceClient.getAll())
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.